cmd/neofs-cli/modules/control/shards_set_mode.go

```diff
@@ -99,3 +103,25 @@ func getShardID(cmd *cobra.Command) []byte {
 	common.ExitOnErr(cmd, "incorrect shard ID encoding: %w", err)
 	return raw
 }
+
+func getShardIDList(cmd *cobra.Command) [][]byte {
+	all, _ := cmd.Flags().GetBool(shardAllFlag)
+	if all {
+		return nil
+	}
+
+	sidList, _ := cmd.Flags().GetStringSlice(shardIDFlag)
+	if len(sidList) == 0 {
+		common.ExitOnErr(cmd, "", fmt.Errorf("either --%s or --%s flag must be provided", shardIDFlag, shardAllFlag))
+	}
+
+	res := make([][]byte, 0, len(sidList))
+	for i := range sidList {
+		raw, err := base58.Decode(sidList[i])
+		common.ExitOnErr(cmd, "incorrect shard ID encoding: %w", err)
+
+		res = append(res, raw)
```

Comment:
can we check their uniqueness here? and mb in the control/shard services

Reply:
fixed